contracts/utils/math/Math.sol

```diff
@@ -67,6 +67,8 @@ library Math {
 
             // Handle non-overflow cases, 256 by 256 division.
             if (prod1 == 0) {
+                // Will revert if denominator == 0. It is not possible to disable the check for division by zero using the unchecked block.
```

Comment:
I think this expresses the message better:

```suggestion
                // Will revert if denominator == 0. The surrounding unchecked block does not change this fact.
```

Reply:
Agree with the wording change.